src/PharoThings-Devices-HD44780.package/PotLCD.class/instance/write.charMode..st

```diff
@@ -0,0 +1,15 @@
+writing
+write: anHex charMode: aBit
+	| j |
+	j := 1.
+	(Delay forSeconds: 0.0001) wait.
+	"1 miliseconds waiting to prevent write too quickly"
+	(gpioDic at: #modePin) value: aBit.
```

Comment:
here it would be:
```Smalltalk
modePin value: aBit
```
And if it is a charMode in fact would not it be better to name it as a charModePin?
(and I already asked what is charMode?)

Reply:
Yes, it is better.